tools/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/hashicorp/vault-plugin-scaffolding/tools
+
+go 1.17
```

Comment:
1.19 here as well?

Reply:
Good catch. Done in [985f9d4](https://github.com/hashicorp/vault-plugin-auth-azure/pull/65/commits/985f9d49276d7e45f3b267ce282ec10a497d0acc).